### DISCUSSION TITLE
Mouse issues after update to macOS Monterey (M1 Chip)

### DISCUSSION THREAD

**Discussion by KLukan** [brackets-cont/brackets#115] [Q&A]
I upgraded to macOS Monterey the other day and I'm now noticing that when I try to scroll a page of code in Brackets (magic mouse) the scroll is very "jumpy"...sometimes not scrolling at all. Particularly when trying to scroll up the page. Has anyone else noticed this? I've check a bunch of other apps and the issue does not occur.

**Comment by abose** [comment]
Hi @KLukan Would it be possible to post a video or screencast to help with understanding the problem.

Can use https://www.cockos.com/licecap/ to capture gif screenshots.

**Reply by KLukan** [reply]
@abose No problem. I recorded a Loom video, which you can view here:
https://www.loom.com/share/01246aa027c34dd5905e7da9e4a11f5e

Let me know if you have any questions!

**Comment by charlypa** [comment]
@KLukan 
We have modernized brackets. We released it yesterday.  The new editor is Phoenix Code, Now you can download native m1 binaries instead of relying on old Brackets x86 binaries
https://phcode.io/
